### ISSUE TITLE
unhandled error, server is down after nodes creation

### ISSUE THREAD

**Issue by nikage** [supergiant/control#1083]
**Short Summary:**
server logs 
```E1130 19:46:33.569879   26287 portforward.go:237] Error accepting connection on port 53666: accept tcp4 127.0.0.1:53666: accept: too many open files
E1130 19:46:33.569879   26287 portforward.go:237] Error accepting connection on port 53666: accept tcp4 127.0.0.1:53666: accept: too many open files
log: exiting because of error: log: cannot create log: open /var/folders/n_/cphg95td3ds5xr6s9_3qdt0c0000gn/T/main.MacBook-Pro-Nikola.nikolam.log.ERROR.20181130-194633.26287: too many open files
ERRO[2018-11-30T19:46:33+02:00] helm: list releases: 990fb735 cluster: list releases: rpc error: code = Unavailable desc = transport is closing 
```
exit status 2
**Steps to Reproduce:**

**Expected Results:**

**Actual Results:**

**Dev Info:** (add links to log files)
1. Output of `go version`:
2. Commit hash or release tag used (`git log`):
3. Number of Masters and Nodes:
4. Cloud Provider:
5. Logs from `/tasks/{task_id}/logs/ws` UI/API endpoint for involved clusters:
6. Log files from `/tmp`, on the machine Control is running on (for example: `/tmp/314c2c2c-62b7-461b-8022-04dabeb0d19d.log`):

**Comment by gopherstein** [comment]
Please close this if it is a duplicate.

**Comment by saheienko** [comment]
@eldridgeh that's the result of the #916 issue.

Here upstream library tries to log `to many open files` error but fails to open a log file and exits with status 2.

@gopherstein Is it ok for control to keep working on this error?

**Comment by stgleb** [comment]
Well, I've checked a connections inside master node

```
lsof|grep 10250|wc -l
14483
```
And this number of connections between kube-proxy and kubelet is growing.

https://github.com/kubernetes/kubernetes/issues/67382

looks like fix has been merged not a long time ago

**Comment by saheienko** [comment]
@stgleb how does this related to the control issue?

**Comment by gopherstein** [comment]
This has been resolved with #916
